pom.xml

```diff
@@ -376,5 +383,11 @@
             <version>1.2.6</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.extensions.webconsolesecurityprovider</artifactId>
+            <version>1.2.4</version>
+            <scope>test</scope>
```

Comment:
Apologies for the late review @enapps-enorman , but is the `test` scope the right one here?

Reply:
IMHO yes, as the API is part of Felix. The Sling Web Console Security Provider is only one implementation which is used in ITs.

Reply:
Ah, got it. Thanks @kwin